rust/worker/src/server.rs

```diff
@@ -131,6 +133,13 @@ impl WorkerServer {
             }
         };
 
+        #[cfg(debug_assertions)]
+        {
+            if segment_uuid == Uuid::nil() {
+                panic!("Invalid Segment UUID (throwing panic for testing)");
+            }
+        }
```

Comment:
is there a better way to intentionally force a panic from within a test?

Reply:
I think the second option that makes sense to em rather than a somewhat strange invariant that this nil means panic in debug state